docs/html-css/week-1/lesson.md

```diff
@@ -33,19 +34,38 @@ Learning objectives for this lesson can be found on the [Learning Objectives pag
 
 ## HTML/CSS Project
 
-In today's class, we will begin adapting styles on this Bikes for Refugees example website.
+In today's class, we will begin adapting styles on the [Bikes for Refugees](https://github.com/CodeYourFuture/bikes-for-refugees) project.
 We'll review some of the HTML/CSS basics you already encountered during your application process and learn some new concepts.
-By the end of the third lesson, we will have worked together to improve the example site on the left screenshot below so that it looks like the right screenshot below.
+Fork the repository to your personal account and then clone the repository.
```

Comment:
This was updated since we only work together on the bikes for refugees project in lesson 1, and it's lesson 2's homework to finish it.

Also, there are screenshots in the bikes for refugees repo so these screenshots were removed to stop them from going stale, and keep the single source of truth screenshots in the bikes for refugees repo.